### PR TITLE
Enable desynths with skill rank mismatches meeting certain criteria.

### PR DESCRIPTION
**Pull request by Dirk-Dieters**
Continuing the work of #1306, This is the first step of verifying excluded recipes following the synthesis database rewrite. Per [IBM2431's comment](https://github.com/project-topaz/topaz/pull/1306#issuecomment-717001439), I have activated and assigned levels to recepies with the following criteria:

1. Desynth with FFXIAH listing the skill level as 255
2. BGWiki lists a specific level (no approximations or verification needed flags)
3. Recipe and results match

No assumptions were made based on similar recipes (various gem rings and earrings for example), and no other data was considered.

Recipes with a specific BGWiki skill level but a mismatch in yield or recipe had their skill level set, but remained commented out and had comment added at the end of the line.

**_I affirm:_**
- [X] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [X] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -633,18 +633,18 @@ INSERT INTO `synth_recipes` VALUES (5508,0,0,115,70,0,0,0,0,0,0,4098,4240,1346,4
 -- INSERT INTO `synth_recipes` VALUES (9912,1,0,255,0,0,0,0,0,0,0,4100,4242,17523,0,0,0,0,0,0,0,651,711,711,711,1,1,1,1,'Quarterstaff (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9913,1,0,255,0,0,0,0,0,0,0,4100,4242,12290,0,0,0,0,0,0,0,708,650,708,708,1,1,2,2,'Maple Shield (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9914,1,0,255,0,0,0,0,0,0,0,4100,4242,17347,0,0,0,0,0,0,0,917,714,714,714,1,1,1,1,'Piccolo (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9915,1,0,255,0,0,0,0,0,0,0,4100,4242,17590,0,0,0,0,0,0,0,771,771,771,771,1,1,1,1,'Primate Staff (desynth)');
+INSERT INTO `synth_recipes` VALUES (9915,1,0,99,0,0,0,0,0,0,0,4100,4242,17590,0,0,0,0,0,0,0,771,771,771,771,1,1,1,1,'Primate Staff (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: HQ2: Darksteel Nugget x6

File: sql/synth_recipes.sql [hunk 2]
```diff
@@ -633,18 +633,18 @@ INSERT INTO `synth_recipes` VALUES (5508,0,0,115,70,0,0,0,0,0,0,4098,4240,1346,4
...
 -- INSERT INTO `synth_recipes` VALUES (9916,1,0,255,0,0,0,0,0,0,0,4100,4242,17101,0,0,0,0,0,0,0,720,720,720,720,2,2,2,2,'Mythic Pole (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9917,1,0,255,0,0,0,0,0,0,0,4100,4242,12359,0,0,0,0,0,0,0,1234,1234,1234,1234,4,6,6,6,'Hickory Shield (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9918,1,0,255,0,0,0,0,0,0,0,4100,4242,12292,0,0,0,0,0,0,0,717,717,651,651,1,2,1,1,'Mahogany Shield (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9919,1,0,255,0,0,0,0,0,0,0,4100,4242,17292,0,0,0,0,0,0,0,715,718,718,718,1,1,1,1,'Long Boomerang (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9920,1,0,255,0,0,0,0,0,0,0,4100,4242,17052,0,0,0,0,0,0,0,710,847,847,847,1,1,1,1,'Chestnut Wand (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9921,1,0,255,0,0,0,0,0,0,0,4100,4242,17388,0,0,0,0,0,0,0,707,820,820,820,1,1,1,1,'Fastwater F. Rod (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9922,1,0,255,0,0,0,0,0,0,0,4100,4242,17219,0,0,0,0,0,0,0,932,717,1226,1226,1,1,6,6,'Arbalest (desynth)');
+INSERT INTO `synth_recipes` VALUES (9922,1,0,59,0,0,0,0,0,0,0,4100,4242,17219,0,0,0,0,0,0,0,932,717,1226,1226,1,1,6,6,'Arbalest (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: HQ2: Mythril Nugget x4

File: sql/synth_recipes.sql [hunk 3]
```diff
@@ -633,18 +633,18 @@ INSERT INTO `synth_recipes` VALUES (5508,0,0,115,70,0,0,0,0,0,0,4098,4240,1346,4
...
 -- INSERT INTO `synth_recipes` VALUES (9923,1,0,255,0,0,0,0,0,0,0,4100,4242,17100,0,0,0,0,0,0,0,719,719,719,719,1,1,2,2,'Ebony Pole (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9924,1,0,255,0,0,0,0,0,0,0,4100,4242,16839,0,0,0,0,0,0,0,715,822,1228,1228,1,1,6,6,'Partisan (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (9925,1,0,255,0,0,0,0,0,0,0,4100,4242,17354,0,0,0,0,0,0,0,927,927,927,927,2,2,2,2,'Harp (desynth)');
--- INSERT INTO `synth_recipes` VALUES (9926,1,0,255,0,0,0,0,0,0,0,4100,4242,17348,0,0,0,0,0,0,0,917,716,716,716,1,1,1,1,'Traversiere (desynth)');
+INSERT INTO `synth_recipes` VALUES (9926,1,0,37,0,0,0,0,0,0,0,4100,4242,17348,0,0,0,0,0,0,0,917,716,716,716,1,1,1,1,'Traversiere (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: HQ1: Parchment x1

File: sql/synth_recipes.sql [hunk 4]
```diff
@@ -1351,24 +1351,24 @@ INSERT INTO `synth_recipes` VALUES (15514,0,0,0,118,0,0,0,0,0,0,4096,4238,687,87
...
 -- INSERT INTO `synth_recipes` VALUES (19956,1,0,0,255,0,0,0,0,0,0,4100,4242,566,0,0,0,0,0,0,0,649,649,649,649,1,1,1,1,'Goblin Cup (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19957,1,0,0,255,0,0,0,0,0,0,4100,4242,16796,0,0,0,0,0,0,0,817,1226,717,717,3,10,1,1,'Mythril Zaghnal (desynth)');
+INSERT INTO `synth_recipes` VALUES (19957,1,0,0,61,0,0,0,0,0,0,4100,4242,16796,0,0,0,0,0,0,0,817,1226,717,717,3,10,1,1,'Mythril Zaghnal (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19958,1,0,0,255,0,0,0,0,0,0,4100,4242,16544,0,0,0,0,0,0,0,649,649,649,649,1,1,1,1,'Ryl.Arc. Sword (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19959,1,0,0,255,0,0,0,0,0,0,4100,4242,16719,0,0,0,0,0,0,0,649,649,649,649,2,2,2,2,'Moth Axe (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19960,1,0,0,255,0,0,0,0,0,0,4100,4242,16783,0,0,0,0,0,0,0,817,651,715,715,3,1,1,1,'Plantreaper (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19961,1,0,0,255,0,0,0,0,0,0,4100,4242,14051,0,0,0,0,0,0,0,651,654,654,654,1,1,1,1,'Alumine Moufles (desynth)');
+INSERT INTO `synth_recipes` VALUES (19961,1,0,0,59,0,0,0,0,0,0,4100,4242,14051,0,0,0,0,0,0,0,651,654,654,654,1,1,1,1,'Alumine Moufles (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: HQ1: Iron Ingot x1

File: sql/synth_recipes.sql [hunk 5]
```diff
@@ -3185,15 +3185,15 @@ INSERT INTO `synth_recipes` VALUES (45506,0,0,0,0,70,0,113,0,0,0,4099,4241,8722,
...
--- INSERT INTO `synth_recipes` VALUES (49944,1,0,0,0,0,0,255,0,0,0,4100,4242,13703,0,0,0,0,0,0,0,850,852,851,851,1,3,1,1,'Brigandine (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49945,1,0,0,0,0,0,255,0,0,0,4100,4242,14182,0,0,0,0,0,0,0,816,851,849,821,1,2,1,1,'Errant Pigaches (desynth)');
+INSERT INTO `synth_recipes` VALUES (49944,1,0,0,0,0,0,57,0,0,0,4100,4242,13703,0,0,0,0,0,0,0,850,852,851,851,1,3,1,1,'Brigandine (desynth)');
+INSERT INTO `synth_recipes` VALUES (49945,1,0,0,0,0,0,91,0,0,0,4100,4242,14182,0,0,0,0,0,0,0,816,851,849,821,1,2,1,1,'Errant Pigaches (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49946,1,0,0,0,0,0,255,0,0,0,4100,4242,12442,0,0,0,0,0,0,0,850,651,651,651,1,1,1,1,'Studded Bandana (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49947,1,0,0,0,0,0,255,0,0,0,4100,4242,12993,0,0,0,0,0,0,0,850,848,848,848,1,1,1,1,'Sandals (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49948,1,0,0,0,0,0,255,0,0,0,4100,4242,13595,0,0,0,0,0,0,0,820,863,863,863,1,1,1,1,'Coeurl Mantle (desynth)');
+INSERT INTO `synth_recipes` VALUES (49948,1,0,0,0,0,0,85,0,0,0,4100,4242,13595,0,0,0,0,0,0,0,820,863,863,863,1,1,1,1,'Coeurl Mantle (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql:3196]: BGWiki seems confused about HQ yields on their page; I'm fine with these.

File: sql/synth_recipes.sql [hunk 6]
```diff
@@ -3605,18 +3605,18 @@ INSERT INTO `synth_recipes` VALUES (55524,0,0,0,0,0,0,0,117,0,0,4099,4241,9003,9
...
 -- INSERT INTO `synth_recipes` VALUES (59942,1,0,0,0,0,0,0,255,0,0,4100,4242,13108,0,0,0,0,0,0,0,850,887,887,887,1,1,1,1,'Coral Gorget (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59943,1,0,0,0,0,0,0,255,0,0,4100,4242,13464,0,0,0,0,0,0,0,864,902,902,902,1,1,1,1,'Demon\'s Ring (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59943,1,0,0,0,0,0,0,70,0,0,4100,4242,13464,0,0,0,0,0,0,0,864,902,902,902,1,1,1,1,'Demon\'s Ring (desynth)'); --yield mismatch
 -- INSERT INTO `synth_recipes` VALUES (59944,1,0,0,0,0,0,0,255,0,0,4100,4242,12436,0,0,0,0,0,0,0,850,866,866,866,1,2,2,2,'Dragon Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59945,1,0,0,0,0,0,0,255,0,0,4100,4242,13458,0,0,0,0,0,0,0,896,896,896,896,1,1,1,1,'Scorpion Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59946,1,0,0,0,0,0,0,255,0,0,4100,4242,17257,0,0,0,0,0,0,0,893,893,893,893,1,1,1,1,'Bandit\'s Gun (desynth)');
+INSERT INTO `synth_recipes` VALUES (59945,1,0,0,0,0,0,0,55,0,0,4100,4242,13458,0,0,0,0,0,0,0,896,896,896,896,1,1,1,1,'Scorpion Ring (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: [BGWiki is claiming this Scorpion Ring desynth is 60](https://www.bg-wiki.com/bg/Scorpion_Ring). Both [FFXIAH](https://www.ffxiah.com/recipes/555) and BGWiki say the _synth_ is 60.

File: sql/synth_recipes.sql [hunk 7]
```diff
@@ -4170,28 +4170,28 @@ INSERT INTO `synth_recipes` VALUES (65512,0,0,0,0,0,0,0,0,118,0,4096,4238,9002,9
...
+INSERT INTO `synth_recipes` VALUES (69918,1,0,0,0,0,0,0,0,51,0,4100,4242,16430,0,0,0,0,0,0,0,937,649,649,649,1,1,1,1,'Acid Claws (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (69919,1,0,0,0,0,0,0,0,255,0,4100,4242,1624,0,0,0,0,0,0,0,933,933,933,933,10,10,10,10,'Bugbear Mask (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (69920,1,0,0,0,0,0,0,0,255,0,4100,4242,16858,0,0,0,0,0,0,0,5306,1226,1226,1226,1,10,10,10,'Sacred Lance (desynth)');
--- INSERT INTO `synth_recipes` VALUES (69921,1,0,0,0,0,0,0,0,255,0,4100,4242,16860,0,0,0,0,0,0,0,4154,1226,715,715,1,10,2,2,'Holy Lance (desynth)');
--- INSERT INTO `synth_recipes` VALUES (69922,1,0,0,0,0,0,0,0,255,0,4100,4242,17041,0,0,0,0,0,0,0,4154,716,1226,653,1,1,10,3,'Holy Mace (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (69921,1,0,0,0,0,0,0,0,83,0,4100,4242,16860,0,0,0,0,0,0,0,4154,1226,715,715,1,10,2,2,'Holy Lance (desynth)'); --yield mismatch
+INSERT INTO `synth_recipes` VALUES (69922,1,0,0,0,0,0,0,0,55,0,4100,4242,17041,0,0,0,0,0,0,0,4154,716,1226,653,1,1,10,3,'Holy Mace (desynth)');
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: Hrm. BGWiki is claiming HQ3 of Mythril Ingot x2, while the current quantity came from [FFXIAH claiming Mythril Ingot x3](https://www.ffxiah.com/recipes/423). Dunno how we want to proceed, or if we want to keep excluding the recipe for further verification.

File: sql/synth_recipes.sql [hunk 8]
```diff
@@ -1296,7 +1296,7 @@ INSERT INTO `synth_recipes` VALUES (15514,0,0,0,118,0,0,0,0,0,0,4096,4238,687,87
 -- INSERT INTO `synth_recipes` VALUES (15532,0,0,255,255,0,0,0,0,0,0,4096,4238,4077,4080,9246,16942,0,0,0,0,21656,21657,21657,21657,1,1,1,1,'Dyrnwyn');
 -- INSERT INTO `synth_recipes` VALUES (15533,0,1998,0,255,0,0,0,255,0,0,4102,4244,9005,9062,9249,9831,9844,9845,9845,9864,21582,21583,21584,21584,1,1,1,1,'Etoile Knife');
 -- INSERT INTO `synth_recipes` VALUES (19901,1,0,0,255,0,0,0,0,0,0,4100,4242,15288,0,0,0,0,0,0,0,672,850,850,850,1,4,4,4,'Pellet Belt (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19902,1,0,0,255,0,0,0,0,0,0,4100,4242,12299,0,0,0,0,0,0,0,715,649,649,649,1,3,3,3,'Aspis (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (19902,1,0,0,18,0,0,0,0,0,0,4100,4242,12299,0,0,0,0,0,0,0,715,649,649,649,1,3,3,3,'Aspis (desynth)'); --Yeild mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: For _yield_ mismatches (_not ingredients_), I'd be okay with going with just BGWiki unless FFXIAH deliberately contradicts.

FFXIAH only recorded what it saw and I don't think it recorded/displays different HQ tiers properly. So if it only saw an [HQ of Bronze Ingot x3](https://www.ffxiah.com/recipes/3199), _I'd_ be okay with [BGWiki's claims that the Bronze Ingot x3 is HQ3, and filling in their claimed HQ1 and HQ2 as appropriate](https://www.bg-wiki.com/bg/Aspis).

But in this case, FFXIAH doesn't _deliberately_ contradict BGWiki by having all tiers filled out with conflicting information. It only has two tiers filled out with mislabeled "tier levels".

File: sql/synth_recipes.sql [hunk 9]
```diff
@@ -1351,24 +1351,24 @@ INSERT INTO `synth_recipes` VALUES (15514,0,0,0,118,0,0,0,0,0,0,4096,4238,687,87
...
 -- INSERT INTO `synth_recipes` VALUES (19959,1,0,0,255,0,0,0,0,0,0,4100,4242,16719,0,0,0,0,0,0,0,649,649,649,649,2,2,2,2,'Moth Axe (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19960,1,0,0,255,0,0,0,0,0,0,4100,4242,16783,0,0,0,0,0,0,0,817,651,715,715,3,1,1,1,'Plantreaper (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19961,1,0,0,255,0,0,0,0,0,0,4100,4242,14051,0,0,0,0,0,0,0,651,654,654,654,1,1,1,1,'Alumine Moufles (desynth)');
+INSERT INTO `synth_recipes` VALUES (19961,1,0,0,59,0,0,0,0,0,0,4100,4242,14051,0,0,0,0,0,0,0,651,654,654,654,1,1,1,1,'Alumine Moufles (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19962,1,0,0,255,0,0,0,0,0,0,4100,4242,16433,0,0,0,0,0,0,0,649,649,649,649,1,1,1,1,'Lgn. Knuckles (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19963,1,0,0,255,0,0,0,0,0,0,4100,4242,2161,0,0,0,0,0,0,0,649,649,649,649,2,2,2,2,'Troll Vambrace (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (19963,1,0,0,255,0,0,0,0,0,0,4100,4242,2161,0,0,0,0,0,0,0,649,649,649,649,2,2,2,2,'Troll Vambrace (desynth)'); --yield mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: I suspect you're going to do another PR for yields, so I won't comment on those further, and just stick to levels for excluded recipes~

(So people who _aren't_ the two of us know what's on BGWiki)

Smithing 79

File: sql/synth_recipes.sql [hunk 10]
```diff
@@ -1351,24 +1351,24 @@ INSERT INTO `synth_recipes` VALUES (15514,0,0,0,118,0,0,0,0,0,0,4096,4238,687,87
...
 -- INSERT INTO `synth_recipes` VALUES (19966,1,0,0,255,0,0,0,0,0,0,4100,4242,16452,0,0,0,0,0,0,0,654,654,654,654,1,1,1,1,'Misericorde (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19967,1,0,0,255,0,0,0,0,0,0,4100,4242,16412,0,0,0,0,0,0,0,894,1226,1226,1226,1,6,6,6,'Mythril Claws (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19968,1,0,0,255,0,0,0,0,0,0,4100,4242,17487,0,0,0,0,0,0,0,937,937,937,937,1,1,1,1,'Corrosive Claws (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19969,1,0,0,255,0,0,0,0,0,0,4100,4242,2162,0,0,0,0,0,0,0,651,652,652,652,1,1,1,1,'Mamool Ja Helmet (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19970,1,0,0,255,0,0,0,0,0,0,4100,4242,16589,0,0,0,0,0,0,0,715,852,651,651,1,1,5,5,'Two-Hand. Sword (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19971,1,0,0,255,0,0,0,0,0,0,4100,4242,16707,0,0,0,0,0,0,0,1235,719,1228,1228,6,6,8,8,'Bhuj (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (19971,1,0,0,255,0,0,0,0,0,0,4100,4242,16707,0,0,0,0,0,0,0,1235,719,1228,1228,6,6,8,8,'Bhuj (desynth)'); --yield mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: Smithing 95

File: sql/synth_recipes.sql [hunk 11]
```diff
@@ -3149,8 +3149,8 @@ INSERT INTO `synth_recipes` VALUES (45506,0,0,0,0,70,0,113,0,0,0,4099,4241,8722,
 -- INSERT INTO `synth_recipes` VALUES (49901,1,0,0,0,0,0,255,0,0,0,4100,4242,13588,0,0,0,0,0,0,0,857,820,820,820,1,1,1,1,'Dhalmel Mantle (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49902,1,0,0,0,0,0,255,0,0,0,4100,4242,16385,0,0,0,0,0,0,0,715,850,850,850,1,2,2,2,'Cesti (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49903,1,0,0,0,0,0,255,0,0,0,4100,4242,13918,0,0,0,0,0,0,0,861,820,851,1122,1,1,1,1,'Tiger Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49904,1,0,0,0,0,0,255,0,0,0,4100,4242,16388,0,0,0,0,0,0,0,715,850,852,853,1,2,1,1,'Himantes (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49905,1,0,0,0,0,0,255,0,0,0,4100,4242,14317,0,0,0,0,0,0,0,1680,1629,1629,1629,1,2,2,2,'Barone Cosciales (desynth)');
+INSERT INTO `synth_recipes` VALUES (49904,1,0,0,0,0,0,52,0,0,0,4100,4242,16388,0,0,0,0,0,0,0,715,850,852,853,1,2,1,1,'Himantes (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (49905,1,0,0,0,0,0,255,0,0,0,4100,4242,14317,0,0,0,0,0,0,0,1680,1629,1629,1629,1,2,2,2,'Barone Cosciales (desynth)'); --yield mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: Leathercraft 90

File: sql/synth_recipes.sql [hunk 12]
```diff
@@ -3161,14 +3161,14 @@ INSERT INTO `synth_recipes` VALUES (45506,0,0,0,0,70,0,113,0,0,0,4099,4241,8722,
 -- INSERT INTO `synth_recipes` VALUES (49913,1,0,0,0,0,0,255,0,0,0,4100,4242,13081,0,0,0,0,0,0,0,817,851,851,851,1,1,1,1,'Leather Gorget (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49914,1,0,0,0,0,0,255,0,0,0,4100,4242,12954,0,0,0,0,0,0,0,848,651,651,651,1,2,2,2,'Studded Boots (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (49915,1,0,0,0,0,0,255,0,0,0,4100,4242,12824,0,0,0,0,0,0,0,817,850,850,850,6,2,2,2,'Leather Trousers (desynth)');
--- INSERT INTO `synth_recipes` VALUES (49916,1,0,0,0,0,0,255,0,0,0,4100,4242,13594,0,0,0,0,0,0,0,817,856,856,856,1,5,5,5,'Rabbit Mantle (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (49916,1,0,0,0,0,0,255,0,0,0,4100,4242,13594,0,0,0,0,0,0,0,817,856,856,856,1,5,5,5,'Rabbit Mantle (desynth)'); --yield mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: Leathercraft 7

File: sql/synth_recipes.sql [hunk 13]
```diff
@@ -3581,21 +3581,21 @@ INSERT INTO `synth_recipes` VALUES (55524,0,0,0,0,0,0,0,117,0,0,4099,4241,9003,9
...
 -- INSERT INTO `synth_recipes` VALUES (59927,1,0,0,0,0,0,0,255,0,0,4100,4242,13922,0,0,0,0,0,0,0,850,886,902,902,1,1,1,1,'Demon Helm (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,255,0,0,4100,4242,13442,0,0,0,0,0,0,0,864,888,888,888,1,1,1,1,'Shell Ring (desynth)');
+INSERT INTO `synth_recipes` VALUES (59928,1,0,0,0,0,0,0,7,0,0,4100,4242,13442,0,0,0,0,0,0,0,864,888,888,888,1,1,1,1,'Shell Ring (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (59929,1,0,0,0,0,0,0,255,0,0,4100,4242,13461,0,0,0,0,0,0,0,881,1226,653,653,1,4,1,1,'Carapace Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59930,1,0,0,0,0,0,0,255,0,0,4100,4242,12455,0,0,0,0,0,0,0,852,852,852,852,1,1,1,1,'Beetle Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59931,1,0,0,0,0,0,0,255,0,0,4100,4242,14305,0,0,0,0,0,0,0,1629,1629,1629,1629,1,1,1,1,'Dragon Subligar (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59930,1,0,0,0,0,0,0,28,0,0,4100,4242,12455,0,0,0,0,0,0,0,852,852,852,852,1,1,1,1,'Beetle Mask (desynth)'); --yield mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql:3597]: Props for _not_ falling for the "Bonecraft Kit 30" trap that some BGWiki editor did for the synth. 😉

File: sql/synth_recipes.sql [hunk 14]
```diff
@@ -3605,18 +3605,18 @@ INSERT INTO `synth_recipes` VALUES (55524,0,0,0,0,0,0,0,117,0,0,4099,4241,9003,9
...
--- INSERT INTO `synth_recipes` VALUES (59943,1,0,0,0,0,0,0,255,0,0,4100,4242,13464,0,0,0,0,0,0,0,864,902,902,902,1,1,1,1,'Demon\'s Ring (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59943,1,0,0,0,0,0,0,70,0,0,4100,4242,13464,0,0,0,0,0,0,0,864,902,902,902,1,1,1,1,'Demon\'s Ring (desynth)'); --yield mismatch
 -- INSERT INTO `synth_recipes` VALUES (59944,1,0,0,0,0,0,0,255,0,0,4100,4242,12436,0,0,0,0,0,0,0,850,866,866,866,1,2,2,2,'Dragon Mask (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59945,1,0,0,0,0,0,0,255,0,0,4100,4242,13458,0,0,0,0,0,0,0,896,896,896,896,1,1,1,1,'Scorpion Ring (desynth)');
--- INSERT INTO `synth_recipes` VALUES (59946,1,0,0,0,0,0,0,255,0,0,4100,4242,17257,0,0,0,0,0,0,0,893,893,893,893,1,1,1,1,'Bandit\'s Gun (desynth)');
+INSERT INTO `synth_recipes` VALUES (59945,1,0,0,0,0,0,0,55,0,0,4100,4242,13458,0,0,0,0,0,0,0,896,896,896,896,1,1,1,1,'Scorpion Ring (desynth)');
+-- INSERT INTO `synth_recipes` VALUES (59946,1,0,0,0,0,0,0,55,0,0,4100,4242,17257,0,0,0,0,0,0,0,893,893,893,893,1,1,1,1,'Bandit\'s Gun (desynth)'); --yield mismatch
```
Comment (ibm2431) [inline sql/synth_recipes.sql]: BGWiki is claiming 52.

**Review by ibm2431 — COMMENTED** [review]
Safe and conservative changes, just what I wanna see~!

I only reviewed / commented on yields for the recipes that'd _no longer_ be excluded. (And the still-excluded Aspis as an explanation of my opinion on yield mismatches).

File: sql/synth_recipes.sql (hunk 7 above)
Comment (Dirk-Dieters) [reply sql/synth_recipes.sql]: Seems reasonable to wait until we have retail confirmation or at least some more sources.

File: sql/synth_recipes.sql (hunk 8 above)
Comment (Dirk-Dieters) [reply sql/synth_recipes.sql]: In that case, that's the course I'll take on the next PR.

**Review by ibm2431 — APPROVED** [review]
